source/mongodump.txt

```diff
@@ -816,6 +833,13 @@ Options
    You cannot use the :option:`--archive <mongodump --archive>` option
    with the :option:`--out <mongodump --out>` option.
 
+   .. note::
+
+      :binary:`~bin.mongodump` performance can improve if the process
+      writes to an archive file. For more information on ``mongodump``
+      performance impacts, see :ref:`Mongostore and Mongodump
+      Considerations <binary-bson-dumps>`. 
```

Comment:
My hesitation in using the blog post was that is it outdated (references the 3.2 release). Is there another link that might be more relevant?

Reply:
Hm, I think that'd be okay since the archive-related points in the blog post are still relevant (we haven't changed that option since its inception) but if you prefer, I could cherry-pick some points from it and we could add it to the page with the `Considerations` section?

Reply:
I added this section in [DOCSP-36537](https://jira.mongodb.org/browse/DOCSP-36537) so the note will now link to the new output format considerations section. Let me know if there are any other changes you would like to see!